addons/snailmail/models/snailmail_letter.py

```diff
@@ -76,7 +76,10 @@ def _compute_display_name(self):
     @api.depends('model', 'res_id')
     def _compute_reference(self):
         for res in self:
-            res.reference = "%s,%s" % (res.model, res.res_id)
+            reference = ''
```

Comment:
Maybe you can also do same for ir.model 
It also has same issue.

Reply:
@sswapnesh thanks and updated for ir.model also :)